libs/api/domains/education/src/lib/graphql/frigg/option.model.ts

```diff
@@ -0,0 +1,15 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+import { FriggValueModel } from './value.model'
+
+@ObjectType()
```

Comment:
Graphql objects must be prefixed with the domain
```suggestion
@ObjectType('EducationFriggOptionModel')
```

Reply:
Fixed